DH_d: [0.089159, 0, 0, 0.10915, 0.09465, 0.0823]
DH_a: [0 ,-0.425 ,-0.39225 ,0 ,0 ,0]
DH_alpha: [1.57, 0, 0, 1.57, -1.57, 0 ]
DH_theta: [j0,j1,j2,j3,j4,j5]
